src/components/Header.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Remove the 'eslint-disable' comment, and add an 'aria-label' attribute to the button in line 33 to improve accessibility.